backend/domain/Enum/GameStateEnum.js

```diff
@@ -1,6 +1,6 @@
 // Enums for states.
 // Can be developed into classes if functionality is required.
-const GameStateEnum = { NIGHTTIME: 1, DAYTIME: 2, INITIALVOTE: 3, TRIAL: 4 };
+const GameStateEnum = { WAITING_LOBBY: 0, NIGHT_TIME: 1, DAY_TIME: 2, INITIAL_VOTE: 3, TRIAL: 4 };
```

Comment:
Added a `WAITING_LOBBY` state as our initial state, so that when the game starts and switches to night, the game state will correctly be `NIGHT_TIME`. Before, the game state was initialised to `NIGHT_TIME`.

Reply:
nice